meta-refkit-industrial/recipes-ros/ros-comm/roscpp_1.11.21.bbappend

```diff
@@ -0,0 +1,5 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
+
+# Can be removed once https://github.com/ros/ros_comm/pull/1105 is in
+# meta-ros.
+SRC_URI_append = " file://roscpp-add-missing-header-for-writev.patch;striplevel=3"
```

Comment:
I remember I had to add `_df-refkit-industrial` suffix to make bbappends compatible with poky tests. Is it still needed?